include/Gosu/Window.hpp

```diff
@@ -34,7 +34,8 @@ namespace Gosu
         //! \param update_interval Interval in milliseconds between two calls to the update member
         //! function.
         Window(unsigned width, unsigned height, bool fullscreen = false,
-            double update_interval = 16.666666, bool resizable = false);
+            double update_interval = 16.666666, bool resizable = false,
+            bool borderless = false);
```

Comment:
At this point I think we should merge fullscreen, resizable, and borderless into a single `unsigned flags` and introduce a `WindowFlags` enum akin to the existing `ImageFlags`.

Reply:
Gosu::WF_FULLSCREEN should have the value 1 for legacy code that passes `true` for this `unsigned` argument.